### ISSUE TITLE
Visual shaping of slurs in viewer with \vshape

### ISSUE THREAD

**Issue by edizionimigliori** [frescobaldi/frescobaldi#1443]
Lilypond [2.23 introduces \vshape](https://lilypond.org/doc/v2.23/Documentation/changes-big-page.html), which shows the control points and polygon for adjusting slur Bezier curves. It would be AMAZING if a visual tool for shaping slurs could be implemented in the PDF/SVG viewer, where dragging control points created by \vshape resulted in proportional changes to the control point values. This would _drastically_ speed up the process of shaping slurs. Thanks for considering!

`{ a1\vshape #'((0 . 0) (0 . 0.5) (0 . 0.9) (0 . 0.4))^( c'1) }`
![example curve](https://lilypond.org/doc/v2.23/Documentation/3c/lily-cff4c1a9.png)

**Comment by Jean-Abou-Samra** [comment]
Yes. [Spontini](https://github.com/paopre/Spontini) was basically developed with that objective, and it works, but it isn't very comfortable on other aspects (in my personal opinion) and I much prefer Frescobaldi.

This sounds like a duplicate of #265, though?
